src/esssans/direct_beam.py

```diff
@@ -7,11 +7,41 @@
 import scipp as sc
 from sciline import Pipeline
 
-from .types import (
-    BackgroundSubtractedIofQ,
-    DirectBeam,
-    WavelengthBands,
-)
+from .types import BackgroundSubtractedIofQ, DirectBeam, WavelengthBands
+
+
+def get_I0(filename: str, q: sc.Variable) -> sc.Variable:
+    """
+    Get the intensity of the I(Q) for the known sample at a specified Q value from a
+    reference file.
+
+    Parameters
+    ----------
+    filename:
+        The name of the reference file.
+    q:
+        The Q value at which to compute the intensity.
+
+    Returns
+    -------
+    :
+        The interpolated intensity of the I(Q) for the requested Q value.
+
+    """
+    from .loki.data import get_path
+
+    data = np.loadtxt(get_path(filename))
```

Comment:
This function will not be usable for anything but test data then? Can we avoid doing path handling here?